sdk/spring/azure-spring-boot-core/src/main/java/com/azure/spring/core/ApplicationId.java

```diff
@@ -15,23 +13,22 @@
     //    sc: for Spring Cloud
     //    sd: for Spring Data
     //    ss: for Spring Streams
+    //    si: for Spring Integration
     //    kv: for Key Vault
     //    sb: for Storage Blobs
     //    sf: for Storage Files
+    //    sq: for Storage Queue
     //    eh: for Event Hub
     //    bus: for Service Bus
     //    cfg: for App Config
     //    cos: for Cosmos
     //    aad: for AAD
     //    b2c: for AAD B2C
-    public static final String VERSION = Optional.of(ApplicationId.class)
-                                                 .map(Class::getPackage)
-                                                 .map(Package::getImplementationVersion)
-                                                 .orElse("unknown");
-    public static final String AZURE_SPRING_KEY_VAULT = "az-sp-kv/" + VERSION;
```

Comment:
what's the reason here to remove VERSION

Reply:
Prefix has moved to the autoconfigution library, because we need to collection the version of packages.

Reply:
why move it to the auto configuration lib? I thought this is so basic and should be part of the core lib?